pkg/mkverification-raw-efi/spec-verification.sh

```diff
@@ -0,0 +1,547 @@
+#!/bin/sh
```

Comment:
How is this different than debug/spec.sh?
And why do we need a variant file?

Reply:
When we install via USB, `spec.sh` reports the total available storage, the sum of the storage devices plus the space of the USB. I created this script to report the total available storage size, excluding the USB space. Additionally, In spec-verification.sh I changed the logical name of the peripherals to the description of the device in lspci. Maybe you are right we do not need the new file. I can include those changes in spec.sh and I can make them conditional depending on the operation we are doing (installation, verification).

Reply:
How do you exclude the USB stick? If this is done in a way where you exclude removable disks (and not the root disk) then it would make sense as a fix to the existing spec.sh

Do you have examples of the name change - is it based on the type in lspci/lsusb? We need to make sure it makes sense for cellular modems etc.

But a lot easier reviewing the above if is is done as a change to spec.sh.

Reply:
OK, I will do it in `spec.sh`.